### PR TITLE
[PDI-16564] UserDefinedJavaClass Step produces multiple inner CDATA t…

### PR DESCRIPTION
**Pull request by stanislau-strelchanka**
[PDI-16564] UserDefinedJavaClass Step produces multiple inner CDATA tags in transformation XML

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -475,9 +475,9 @@
     </dependency>
     <dependency>
       <groupId>org.codehaus.woodstox</groupId>
-      <artifactId>woodstox-core-asl</artifactId>
-      <version>${woodstox-core-asl.version}</version>
-      <scope>test</scope>
+      <artifactId>wstx-asl</artifactId>
+      <version>3.2.4</version>
```
Comment (kurtwalker) [inline core/pom.xml]: version should be a property reference

**Comment by stanislau-strelchanka** [comment]
@kurtwalker updated, thanks

**Comment by stanislau-strelchanka** [comment]
@mbatchelor please review

**Comment by stanislau-strelchanka** [comment]
@mbatchelor @pamval @mchen-len-son please review

**Comment by stanislau-strelchanka** [comment]
@mbatchelor @pamval @mchen-len-son please review

**Comment by stanislau-strelchanka** [comment]
@mbatchelor @pamval @mchen-len-son please review

**Comment by stanislau-strelchanka** [comment]
@bmorrise please review

**Comment by stanislau-strelchanka** [comment]
@mbatchelor @pamval @mchen-len-son @bmorrise please review

**Comment by stanislau-strelchanka** [comment]
@mbatchelor @pamval @mchen-len-son @bmorrise please review

**Comment by stanislau-strelchanka** [comment]
@mbatchelor @pamval @mchen-len-son @bmorrise please review

**Comment by stanislau-strelchanka** [comment]
@mbatchelor @pamval @mchen-len-son @bmorrise please review

**Pull request merged**